### ISSUE TITLE
Missing Accessibility Features

### ISSUE THREAD

**Issue by ItsOnlyGame** [ItsOnlyGame/nuxt-ui-components#5]
Many components are missing accessibility features that the developer needs to create themselves.  
It would be nice if these were implemented already in the components themselves.
